src/sentry/integrations/tasks/sync_assignee_outbound_impl.py

```diff
@@ -0,0 +1,47 @@
+from sentry import analytics, features
+from sentry.integrations.services.integration import integration_service
+from sentry.models.integrations.external_issue import ExternalIssue
+from sentry.models.integrations.integration import Integration
+from sentry.models.organization import Organization
+from sentry.models.user import User
+from sentry.silo.base import SiloMode
+from sentry.tasks.base import instrumented_task, retry
+from sentry.users.services.user.service import user_service
+
+
+@instrumented_task(
+    name="sentry.integrations.tasks.sync_assignee_outbound_impl",
```

Comment:
Did this task name change? I don't see this name in any current tasks.

Reply:
yes, in the previous/old module the task was named `sentry.tasks.integrations.sync_assignee_outbound`. In the new task/location it's updated to have the full module name. 

Will this lead to conflicts? It shouldn't (?), because all the new module paths are being added to the celery_imports

Reply:
There shouldn't be any conflicts as you've kept a task with the old name around to handle the deploy boundary. Since we're moving/renaming this task could we skip the `_impl` suffix? We have a different namespace which will give us unique names.